sequencer/test/til/lang.go

```diff
@@ -486,6 +485,6 @@ func (p *parser) parse() (*parsedSet, error) {
 	for u := range users {
 		ps.users = append(ps.users, u)
 	}
-	sort.Strings(ps.users)
+	// sort.Strings(ps.users)
```

Comment:
I'm not sure what's the point of sorting here but I'm wondering if this was causing an issue 🤔

Reply:
This causing some errors for this scenario:
```
CreateAccountDeposit C: 2000 // Idx=256+0=256
CreateAccountDeposit A: 2000 // Idx=256+1=257
CreateAccountDeposit D: 500  // Idx=256+2=258
CreateAccountDeposit B: 500  // Idx=256+3=259
```

Til has reorged the order of created accounts like `ABCD`, but the assgined idx for each account should be followed by created order as above comments.

Reply:
Interesting. Any idea why that was not causing an issue in hermez node? Or is there a chance they might have made a mistake in the implementation and missed it 🤔